pkg/cmd/codespace/code.go

```diff
@@ -12,30 +12,40 @@ func newCodeCmd(app *App) *cobra.Command {
 	var (
 		codespace   string
 		useInsiders bool
+		useWeb      bool
 	)
 
 	codeCmd := &cobra.Command{
 		Use:   "code",
 		Short: "Open a codespace in Visual Studio Code",
 		Args:  noArgsConstraint,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return app.VSCode(cmd.Context(), codespace, useInsiders)
+			return app.VSCode(cmd.Context(), codespace, useInsiders, useWeb)
```

Comment:
This seems fine to me. I wonder how much usage this will get and whether `gh codespace web` might be warranted but we can always introduce that later.